From 851695958a224decffa65ea0c8b6622d19a971ba Mon Sep 17 00:00:00 2001
From: bellard <6490144+bellard@users.noreply.github.com>
Date: Sun, 6 Mar 2022 18:55:04 +0100
Subject: [PATCH] updated test262.conf

---
 test262.conf | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/etc/test262.conf b/etc/test262.conf
index 6fbb5793..bb66cf4f 100644
--- a/etc/test262.conf
+++ b/etc/test262.conf
@@ -51,6 +51,7 @@ testdir=test262/test
 AggregateError
 align-detached-buffer-semantics-with-web-reality
 arbitrary-module-namespace-names=skip
+array-find-from-last=skip
 Array.prototype.at=skip
 Array.prototype.flat
 Array.prototype.flatMap
@@ -66,8 +67,10 @@ BigInt
 caller
 class
 class-fields-private
+class-fields-private-in=skip
 class-fields-public
 class-methods-private
+class-static-block=skip
 class-static-fields-public
 class-static-fields-private
 class-static-methods-private
@@ -89,6 +92,7 @@ default-parameters
 destructuring-assignment
 destructuring-binding
 dynamic-import
+error-cause=skip
 export-star-as-namespace-from-module
 FinalizationGroup=skip
 FinalizationRegistry=skip
@@ -102,10 +106,12 @@ globalThis
 hashbang
 host-gc-required=skip
 import.meta
+import-assertions=skip
 Int16Array
 Int32Array
 Int8Array
 IsHTMLDDA
+json-modules=skip
 json-superset
 legacy-regexp=skip
 let
@@ -116,6 +122,7 @@ numeric-separator-literal
 object-rest
 object-spread
 Object.fromEntries
+Object.hasOwn
 Object.is
 optional-catch-binding
 optional-chaining
@@ -134,8 +141,10 @@ regexp-lookbehind
 regexp-match-indices=skip
 regexp-named-groups
 regexp-unicode-property-escapes
+resizable-arraybuffer=skip
 rest-parameters
 Set
+ShadowRealm=skip
 SharedArrayBuffer
 string-trimming
 String.fromCodePoint
@@ -164,6 +173,7 @@ Symbol.toStringTag
 Symbol.unscopables
 tail-call-optimization=skip
 template
+Temporal=skip
 top-level-await=skip
 TypedArray
 TypedArray.prototype.at=skip
